### PR TITLE
feat: add watermark to branded credentials

### PR DESCRIPTION
**Pull request by wadeking98**
# Summary of Changes

Added ability to put a watermark on a credential card (useful for non-production credential testing)
Part of this change adds the `nonProductionWatermark` attribute to the baseOverlay OCA spec.
image
image

# Related Issues

N/A

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

**Comment by cvarjao** [comment]
@wadeking98, can you rename `nonProductionWatermark` to simply `watermark`, and move to the meta layer as it could be translatable?

**Comment by wadeking98** [comment]
@cvarjao 
it seems to be already translatable but sure I can move it into the meta overlay

**Comment by wadeking98** [comment]
hold off on merging this, I'm running into some issues with how it displays on IOS

**Review by bryce-mcmath — CHANGES_REQUESTED** [review]
Once Clecio's comment is addressed I am good to merge this

File: packages/legacy/core/App/components/misc/CredentialCard11.tsx [hunk 1]
```diff
@@ -429,7 +455,12 @@ const CredentialCard11: React.FC<CredentialCard11Props> = ({
     )
   }
   return overlay.bundle ? (
-    <View style={[styles.container, style, { elevation: elevated ? 5 : 0 }]}>
+    <View
+      onLayout={(event) => {
```
Comment (bryce-mcmath) [inline packages/legacy/core/App/components/misc/CredentialCard11.tsx:463]: What's the delay like before this `onLayout` takes effect?

Comment (wadeking98) [reply packages/legacy/core/App/components/misc/CredentialCard11.tsx:463]: the delay is maybe a quarter second

Comment (bryce-mcmath) [reply packages/legacy/core/App/components/misc/CredentialCard11.tsx:463]: So there's a quarter second with no watermark? I wonder if there's another way to do it...

**Comment by thiagoromanos** [comment]
@wadeking98 , will this watermark be implemented on CredentialCard 1.0 or it will be exclusive to version 1.1?

**Pull request closed without merging**